.github/workflows/windows.yml

```diff
@@ -51,8 +51,12 @@ jobs:
         vcvars:
           - 'vcvars32.bat'
           - 'vcvars64.bat'
+        # The 'shared-' prefix is no longer needed (i.e., both 'shared-pgo' and 'pgo' resolve to the
+        # same profile). However, we're double-publishing under both names during the transition
+        # period.
         profile:
           - 'shared-pgo'
+          - 'pgo'
```

Comment:
I opted to just run the build twice (even though it will produce the same artifact). I considered running the build once, then copying over the artifact, but candidly I wasn't sure if that would break other assumptions in the pipeline. For example, this job publishes under the name `name: ${{ matrix.py }}-${{ matrix.vcvars }}-${{ matrix.profile }}`, so that would now be "incorrect" for the shared profiles.

Reply:
A problem with this is PGO is non-deterministic. So the builds won't be bit-fit-bit identical.

Having lived through non-deterministic builds in Firefox due to bugs in MSVC's PGO implementation, I am sensitive to having avoidable build variance.

I would strongly prefer to publish the same binary asset under different names.

Let's change the official profile to `pgo` in the building interface. But either change the release publishing code to upload a 2nd copy. Or change the building to emit multiple `.tar.zst` artifacts. I _think_ the release automation will _just work_ if there are multiple `.tar.zst` in the same GitHub Actions artifact. This is how we used to do it until recently.

Reply:
Sounds good, can do. I think the latter will be more straightforward (change `cpython-windows/build.py` to emit two artifacts).

Reply:
(This makes a lot of sense.)